src/site/_release-notes/_3.x.x.adoc

```diff
@@ -20,6 +20,11 @@
 This release contains...
 
 
+[#release-notes-3-x-x-added]
+=== Added
+
+* Add various DSLs for dependency injection. This includes: - `DI.FactoryBuilder` and related builder classes for configuring a `ConfigurableInstanceFactory`. - Repeatable `@TestBinding` test annotation for specifying simple bindings in tests. - `@LegacyLoggerContextSource` test annotation for specifying v1-style configuration files. - Test extension updates to improve compatibility of different annotations. (https://github.com/apache/logging-log4j2/pull/2147[2147])
```

Comment:
As I remarked above: the formatting is terrible.

Reply:
Agreed; yikes

Reply:
Done.